packages/capabilities/src/provider.js

```diff
@@ -11,8 +11,12 @@
 import { capability, DID, literal, struct } from '@ucanto/validator'
 import { equalWith, fail, equal } from './utils.js'
 
-export const StorageProvider = literal(
-  'did:web:web3.storage:providers:w3up-alpha'
+export const Web3StorageId = literal('did:web:web3.storage').or(
+  literal('did:web:staging.web3.storage')
+)
```

Comment:
Nit: Can we keep it simpler by saying it's `did:web` ? I'm not convinced locking down possible options at the type level is going to be of advantage.